ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -283,35 +283,69 @@ FAN_SS_DRSM8X:
   - DRSM8X
 
 FAN_EDPM_ZHM_ZTM_LPS:
-  id: "16288085548559433728"
   description: "Exhaust fan with sensors (temperature, humidity and low pressure)."
   is_canonical: true
   implements:
   - FAN
   - EDPM
   - ZHM
   - ZTM
-  uses:
-  - run_command
-  - run_status
+  opt_uses:
+  - exhaust_fan_run_command
```

Comment:
should be run command and run status; the exhaust fan piece is irrelevant to the type.

Reply:
Changed to FAN_SS_DPM_ZHM_ZTM with run_command and run_status, exhaust_air prefix has been removed as the device is handling a single element e.g. air only.